ingest/src/main/java/com/microsoft/azure/kusto/ingest/QueuedIngestClient.java

```diff
@@ -269,9 +278,8 @@ public IngestionResult ingestFromResultSet(ResultSetSourceInfo resultSetSourceIn
     }
 
     @Override
-    protected String emendEndpointUri(URIBuilder existingEndpoint) {
-        existingEndpoint.setHost(INGEST_PREFIX + existingEndpoint.getHost());
-        return existingEndpoint.toString();
+    protected String amendEndpointUri(URIBuilder existingEndpoint) {
+        return generateDmUriSuggestion(existingEndpoint);
```

Comment:
Why create a new method? I usually think 1-line methods are harder to read/maintain.

In this case, the way to emend the URL is by suggesting the DM endpoint, whereas in the `StreamingIngestClient` we do that by suggesting the Engine endpoint.

Reply:
Actually I now see that you're reusing this method, so in that context this makes sense.

Reply:
So I can close this?

Reply:
Closed!